package.json

```diff
@@ -20,6 +20,7 @@
     "test:automation:space": "wdio wdio.conf.js --suite space",
     "test:automation:recents": "wdio wdio.conf.js --suite recents",
     "test:tap": "cross-env TAP=true wdio wdio.conf.js --suite tap",
+    "test:integration": "wdio wdio.conf.js --suite recents ; sleep 60 ; wdio wdio.conf.js --suite multiple ; sleep 60 ; wdio wdio.conf.js --suite oneOnOne ; sleep 60 ; wdio wdio.conf.js --suite space",
```

Comment:
Will the `sleep` command work on windows? Anything in the package scripts should be cross-platform.

Reply:
good catch, let me double-check this and get back to you...